db/migrate/20170221195674_tidy_current_roles.rb

```diff
@@ -21,34 +21,10 @@ def up
   def process_existing(original_name, role, permission_names)
     diff = role.permission_diff permission_names
     return if diff.empty?
-    rename_existing role, original_name
+    SeedHelper.rename_existing_role role, original_name
     create_from_seeds original_name, permission_names
   end
 
-  def rename_existing(role, original_name)
```

Comment:
yeah, I know it's a duplication of 3 methods, but in this case I think it's better (safer)